uber/templates/registration/form.html

```diff
@@ -256,15 +256,13 @@
 <div class="form-group staffing staffing-checked">
     <label class="col-sm-3 control-label">Assigned Depts</label>
     <div class="col-sm-9">
-        {{ macros.checkgroup(attendee, 'assigned_depts') }}
-    </div>
-</div>
-
-<div class="form-group staffing staffing-checked">
-    <label class="col-sm-3 control-label">Trusted in which depts</label>
-    <div class="col-sm-9">
-        {{ macros.checkgroup(attendee, 'trusted_depts') }}
-        <span class="popup">{{ macros.popup_link("../static_views/trustDesc.html", "What does this mean?") }}</span>
+      <div class="form-control-static">
+        {{ macros.checkgroup_opts(
+            'assigned_depts_ids',
+            c.DEPARTMENT_OPTS_WITH_DESC,
+            defaults=attendee.assigned_depts_ids,
+            include_empty_hidden=True) }}
+      </div>
```

Comment:
Am I correct that while you can set which departments someone is in, you cannot yet set their roles in that department?  Including things like actually setting whether or not they're a Dept Head, a Point of Contact, or they're supposed to fill out the Dept Head checklist?

Reply:
Correct. That is what I meant by:
> I haven't created the Department create/edit forms yet.

I should have been more explicit ☹️ 

I'll be adding the rest of the UI in short order.

Reply:
Here's a sneak preview:
<img width="1058" alt="screen shot 2017-10-31 at 1 50 11 am" src="https://user-images.githubusercontent.com/2592431/32209280-e7992c64-bddd-11e7-81ab-de67b7798646.png">

Reply:
For clarification - right now you're calling out Department Heads, Points of Contact and Checklist Admins as somehow separate from Roles.  There's a case to be made for "Checklist Admins" to be a special case, as that has implications for other parts of the codebase, but I can easily envision a non-magfest event having Departments without Department Heads, or where they don't want a separate PoC role, etc.

Should the department system have the flexibility to not define any kind of organizational structure on the users?

Reply:
That is correct. Those three roles were previously handled entirely by department heads, and are heavily referenced throughout the codebase. 

The plan is to eventually remove Department Heads, Points of Contact and Checklist Admins as hardcoded roles, and replace them with dynamically created roles. There's been a lot of discussion about this – both in this pull request, and in the other issues I mentioned in my original comment: 
* https://github.com/magfest/ubersystem/issues/2342#issuecomment-278227786
* https://github.com/magfest/ubersystem/issues/1474#issuecomment-148730153

It boils down to expediency. The amount of code that would have to be rewritten to support dynamic Dept Head roles – including new interface elements – would be prohibitive. The only way to get this change finished in a (somewhat?) timely manner was to keep those roles hardcoded.